packages/fast-components-styles-msft/src/pivot/index.ts

```diff
@@ -0,0 +1,106 @@
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import {
+    applyFocusVisible,
+    applyLocalizedProperty,
+    Direction,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import { ensureBrandNormal, hoverContrast } from "../utilities/colors";
+import outlinePattern from "../patterns/outline";
+import { PivotClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+const styles: ComponentStyles<PivotClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<PivotClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    return {
+        pivot: {
+            position: "relative",
+            color: designSystem.foregroundColor,
+            overflow: "hidden",
+        },
+        pivot_tabList: {
+            display: "flex",
+            boxSizing: "border-box",
+        },
+        pivot_tab: {
+            minHeight: "32px",
+            padding: "0 12px",
+            fontSize: "14px",

```

Comment:
Can we use typography?

Reply:
We should at the very least use a helper to map to the font ramp so that if it changes this gets updated for free.

Reply:
We probably should just remove this from here and add the fontsize to the class that gets attached to the renderprop